test/jasmine/tests/config_test.js

```diff
@@ -477,41 +484,42 @@ describe('config argument', function() {
 
         var gd, mockCopy;
 
-        beforeEach(function(done) {
+        beforeEach(function() {
             gd = createGraphDiv();
             mockCopy = Lib.extendDeep({}, mock);
-            done();
         });
 
         afterEach(destroyGraphDiv);
 
         it('allows axis range entry by default', function() {
-            Plotly.plot(gd, mockCopy.data, {});
-
-            var corner = document.getElementsByClassName('edrag')[0];
-            var cornerBox = corner.getBoundingClientRect();
-            var cornerX = cornerBox.left + cornerBox.width / 2;
-            var cornerY = cornerBox.top + cornerBox.height / 2;
+            Plotly.plot(gd, mockCopy.data, {})
+            .then(function() {
+                var corner = document.getElementsByClassName('edrag')[0];
+                var cornerBox = corner.getBoundingClientRect();
+                var cornerX = cornerBox.left + cornerBox.width / 2;
+                var cornerY = cornerBox.top + cornerBox.height / 2;
 
-            click(cornerX, cornerY);
+                click(cornerX, cornerY);
 
-            var editBox = document.getElementsByClassName('plugin-editable editable')[0];
-            expect(editBox).toBeDefined();
-            expect(editBox.getAttribute('contenteditable')).toBe('true');
+                var editBox = document.getElementsByClassName('plugin-editable editable')[0];
+                expect(editBox).toBeDefined();
+                expect(editBox.getAttribute('contenteditable')).toBe('true');
+            });
         });
 
         it('disallows axis range entry when disabled', function() {
-            Plotly.plot(gd, mockCopy.data, {}, { showAxisRangeEntryBoxes: false });
-
-            var corner = document.getElementsByClassName('edrag')[0];
-            var cornerBox = corner.getBoundingClientRect();
-            var cornerX = cornerBox.left + cornerBox.width / 2;
-            var cornerY = cornerBox.top + cornerBox.height / 2;
+            Plotly.plot(gd, mockCopy.data, {}, { showAxisRangeEntryBoxes: false })
+            .then(function() {
```

Comment:
`done` and `failTest` here and above.

Reply:
Good eye! Done in https://github.com/plotly/plotly.js/pull/5340/commits/7f9073228128857f340dbd8180260aabbe8f7179.